src/inc/FastConv.h

```diff
@@ -50,6 +52,37 @@ class CFastConv
     Error_t flushBuffer(float* pfOutputBuffer);
 
 private:
+    bool m_bIsInitialized = false;
+    int m_iBlockLength;
+    int m_iBlockNum;
+    int m_iCirIdx;
+    int m_iLengthOfIr;
+    int m_iReset;
+    float** m_pfIRMatrix;
+    float* m_pfInputBlock;
+    float* m_pfOutputBlock;
+    float* m_pfBlockConvOuput;
+    float* m_pfConvOuput;
+    CFft::complex_t *m_pfFreqInput;
+    CFft::complex_t *m_pfFreqIr;
+    CFft::complex_t *m_pfFreqConv;
+    float *m_pfRealInput;
+    float *m_pfImagInput;
+    float *m_pfRealIr;
+    float *m_pfImagIr;
+    float *m_pfRealConv;
+    float *m_pfImagConv;
+
+    float *m_pfTimeConv;
+    float *m_pfTimeInput;
+    float *m_pfTimeIr;
```

Comment:
You have a whole lot of stuff in this class, partly because it does both time & frequency domain processing. Only some of these members are used in each mode.
You could consider creating separate classes for each mode, and then having `CFastConv` just act as a wrapper, delegating to the correct one.